### ISSUE TITLE
1.20 compatibility

### ISSUE THREAD

**Issue by ykvox** [TheWii/immersive-soundscapes#8]
Sounds related to biomes are not playing in 1.20, everything related to health, player movements seems unaffected

**Comment by BurgerXXL420** [comment]
Got the same problem, would love to have a fix! Did you by any chance stumble upon a solution yourself?

**Comment by ykvox** [comment]
yes ! i actually updated all the parts that were broken by 1.20 predicates updates give me a minute and i'll send you a fork

**Comment by ykvox** [comment]
https://github.com/YkVox/immersive-soundscapes

do note this release contains a single data/ressourcepack file

**Comment by BurgerXXL420** [comment]
Thanks you are absolutely amazing! Sorry for the late answer didn’t have time to properly test it, but it works just fine!
